core/src/com/po/fuck/model/Constants.java

```diff
@@ -1,51 +1,135 @@
 package com.po.fuck.model;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Properties;
+
 public final class Constants {
-    public static final int SCREEN_WIDTH = 1280;
-    public static final int SCREEN_HEIGHT = 720;
+    static Properties properties = null;
+
+    static <T> T loadConstant(String name, Class<T> cls){
+        if(properties == null){
+            properties = new Properties();
+            try {
+                properties.load(new FileInputStream("../core/src/com/po/fuck/model/.properties"));
+            } catch(IOException io){
+                throw new RuntimeException(io.getMessage());
+            }
```

Comment:
Catching to rethrow is not a good practice.

Reply:
Catching to close an application (throw RuntimeException) is a good way, probably

Reply:
WDYM ?

Reply:
I want to have here: exception -> close application. Because if we cannot load constants, we are not able to continue proper running of the application.

Reply:
But it throws `IOException` already, why to catch ?

Reply:
Because IOException does not terminate the program. Otherwise, we would need to write `throws IOException` and catch it afterwards.

Reply:
I think this is not the best place to handle/rethrow exceptions. Maybe I am wrong.